app/src/main/java/org/fossasia/susi/ai/device/deviceconnect/DeviceConnectFragment.kt

```diff
@@ -182,43 +379,8 @@ class DeviceConnectFragment : Fragment(), IDeviceConnectView {
                     PERMISSIONS_REQUEST_CODE_ACCESS_COARSE_LOCATION)
         } else {
             deviceConnectPresenter.isPermissionGranted(true)
-            Timber.d("ASK PERMISSIONS ELSE")
-        }
-    }
-
-    override fun showLocationIntentDialog() {
-        val dialogBuilder = AlertDialog.Builder(requireContext())
-        dialogBuilder.setView(R.layout.select_dialog_item_material)
-
-        dialogBuilder.setTitle(R.string.location_access)
-        dialogBuilder.setMessage(R.string.location_access_message)
-        dialogBuilder.setPositiveButton(R.string.next) { _, _ ->
-            val callGPSSettingIntent = Intent(Settings.ACTION_LOCATION_SOURCE_SETTINGS)
-            startActivityForResult(callGPSSettingIntent, REQUEST_LOCATION_ACCESS)
+            Timber.e("ASK PERMISSIONS ELSE")
```

Comment:
Why?

Reply:
This was created to know that the else part of the checking part of asking permission got executed.

Reply:
But what you said earlier about timber?

Reply:
Just updated it to d